pages/success.tsx

```diff
@@ -1,10 +1,28 @@
-import { Card, Heading, SmallText } from "@cheapreats/react-ui";
+import { Card, Heading, SmallText, Button } from "@cheapreats/react-ui";
 import React from "react";
 import styled from "styled-components";
 import Image from "next/image";
 import Snowfall from "react-snowfall";
+import { useState, useEffect } from "react";
+import { useRouter } from "next/router";
 
 export const Success: React.VFC = () => {
+  const [orderId, setOrderId] = useState<string>("");
+  const router = useRouter();
+
+  useEffect(() => {
+    const orderIdString: string = router.query.id as string;
```

Comment:
As string is usually something you want to avoid if you can

Reply:
This was used to get around this typescript error, "Type 'string[]' is not assignable to type 'string'." Would this be a case where it is something we can't avoid?

Reply:
Why can't you do [0] to grab the first string in the index since that's what it is? Just becuase the override the type doesn't override the underlying JavaScript structure. This will lead to confusion down the road ;).

Reply:
When I tried getting the element at index 0 I would only get the first char in the order id which made me think that the id should be a string but for some reason its being identified as an array of string by typescript.

Reply:
Ok let's take a look during our meeting. In the meantime I would suggest looking at the typing for Router for the entire application as the answer will probs be there.

Also try removing as string and seeing if the error goes away